pkg/master/reconcilers/lease.go

```diff
@@ -221,7 +221,8 @@ func (r *leaseEndpointReconciler) doReconcile(serviceName string, endpointPorts
 	}
 
 	glog.Warningf("Resetting endpoints for master service %q to %v", serviceName, masterIPs)
-	return r.endpointRegistry.UpdateEndpoints(ctx, e, rest.ValidateAllObjectFunc, rest.ValidateAllObjectUpdateFunc)
+	// Use default options, dry-run doesn't make sense here.
```

Comment:
would even drop the dry-run part. Dry-run=false is the default case. And it's obvious that we want to do a real update there. The sentence confuses more than it helps.